package/Dockerfile-final

```diff
@@ -4,7 +4,7 @@ COPY . /build
 RUN export INSTALL_PATH="/build/usr" && \
     cd /build/libvmi && \
     autoreconf -vif && \
-    ./configure --prefix=$INSTALL_PATH --disable-kvm --disable-file --disable-bareflank --disable-safety-checks && \
+    ./configure --prefix=$INSTALL_PATH --disable-kvm --disable-file --disable-bareflank --disable-safety-checks --enable-vmi-debug && \
```

Comment:
I think the LibVMI submodule needs to be upped to the current master for this to work